app/views/unattended/provisioning_templates/finish/windows_default_finish.erb

```diff
@@ -33,6 +33,7 @@ description: |
   puppet_enabled = !host_param_true?('skip-puppet-setup') && (host_puppet_server.present? || host_param_true?('force-puppet'))
   salt_enabled = host_param('salt_master') ? true : false
   chef_enabled = @host.respond_to?(:chef_proxy) && @host.chef_proxy
+  network_location = host_param('networklocation') ? host_param('networklocation') : 'Private'
```

Comment:
You can provide a default to `host_param`
```suggestion
  network_location = host_param('networklocation', 'Private')
```

This is repeated a few times. I'm only commenting once, but please fix it in the other files too.

Reply:
@ekohl Thanks for reviewing! I was just looking for `host_param` in the templates and I did not find any template that makes use of the default. They are using either the notation above or something like 
```suggestion
  network_location = host_param('networklocation') || 'Private'
```
Readability wise I like that the most, although it kind of makes sense to use the default value if exists. Any thoughts?

Reply:
Internally it already calls `host.host_param(param_name) || default` so your latter suggestion can also work. I'm still leaning to `host_param('networklocation', 'Private')` to match `.fetch()` but we don't implement block support so I can see arguments for both.

Reply:
I have implemented your suggestion. I feel that it looks a bit strange. 

Finally, I think it would be a good idea to unify the notation for all the templates.

Reply:
@ekohl @sbernhard You outvoted me, we'll go with @ekohl 's suggestion :) I hope I did not miss any occurences. Please have a look if anything else needs changing. I suggest to open an issue for unifying the usage of `host_param` and `host_param_true?` within the unattended templates. Any objections?